ui/src/message_popup/playtest/hmtca_experience_page.jsx

```diff
@@ -0,0 +1,203 @@
+import type {QuestionT} from './pairs_scenario.jsx';
+import HMTCAScenarios from './hmtca_scenario.jsx';
+import HMTCAGroupReview from './hmtca_group_review.jsx';
+
+type ResponseT = {
+  choice:string,
+  question:QuestionT,
+  sceneNumber?:number,
+  anonymizedText?:string
+};
+
+
+
+// This is the flow for the HMTCA breakout session
+export default React.createClass({
+  displayName: 'HMTCAExperiencePage',
+
+  propTypes: {
+    query: React.PropTypes.shape({
+      cohort: React.PropTypes.string,
+      p: React.PropTypes.string
+    }).isRequired
+  },
+
+  // User types cohort for team
+  getInitialState() {
+    return {
+      cohortKey: this.props.query.cohort || '',
+      bucketId: HMTCAScenarios.BUCKETS[0].id.toString(),
+      questions: null,
+      sessionId: uuid.v4()
+    };
+  },
+
+  onCohortKeyChanged(e) {
+    this.setState({ cohortKey: e.target.value });
+  },
+
+  onBucketChanged(e) {
+    this.setState({ bucketId: e.target.value });
+  },
+
+  // Making questions from the cohort
+  onStart(e) {
+    e.preventDefault();
+    const {cohortKey, bucketId} = this.state;
+    const allQuestions = HMTCAScenarios.questionsFor(cohortKey, parseInt(bucketId, 10));
+
+    const startQuestionIndex = this.props.query.p || 0; // for testing or demoing
+    const questions = allQuestions.slice(startQuestionIndex);
+    const questionsHash = hash(JSON.stringify(questions));
+    this.setState({
+      questions,
+      questionsHash
+    });
+  },
+
+  onResetSession() {
+    this.setState(this.getInitialState());
+  },
+
+  onLogMessage(type, response) {
+    const {cohortKey, bucketId, sessionId, questionsHash} = this.state;
+    
+    // Watch for a particular message, then add in the applesKey and double-log
+    // it, stripping out all the identifiers from the log message so we can read it
+    // back later safely anonymized.
+    //
+    // The appleKey is built from (cohort, bucket), so that each of those has its own
+    // scene number space (the number is used for ordering and is user-facing).
+    if (type === 'anonymized_apples_to_apples_partial') {
+      Api.logApplesText({
+        applesKey: [cohortKey, bucketId].join(':'),
+        sceneNumber: response.sceneNumber,
+        sceneText: response.question.text,
+        anonymizedText: response.anonymizedText
+      });
+    }
+
+    Api.logEvidence(type, {
+      ...response,
+      sessionId,
+      cohortKey,
+      questionsHash
+    });
+  },
+
+  render() {
+    return (
+      <SessionFrame onResetSession={this.onResetSession}>
+        {this.renderContent()}
+      </SessionFrame>
+    );
+  },
+
+  renderContent() {
+    const {questions} = this.state;
+    if (!questions) return this.renderIntro();
+
+    return <LinearSession
+      questions={questions}
+      questionEl={this.renderQuestionEl}
+      summaryEl={this.renderClosingEl}
+      onLogMessage={this.onLogMessage}
+    />;
+  },
+
+
+  renderIntro() {
```

Comment:
This is a different intro screen, based on the prototype deck.  The value the user picks here gets passed along into the code returning the slides in the `hmtca_scenario.jsx` file.